### PR TITLE
cnao, Create bump cnao on kubevirtci script

### PR DESCRIPTION
**Pull request by oshoval**
This script allows to easily bump CNAO of a specific
provider to a specific CNAO version.

Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: hack/bump-cnao.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+
+set -e
```
Comment (RamLavi) [inline hack/bump-cnao.sh:19]: can we add x?
```suggestion
set -xe
```

**Review by RamLavi — CHANGES_REQUESTED** [review]

File: hack/bump-cnao.sh (hunk 1 above)
Comment (oshoval) [reply hack/bump-cnao.sh:19]: I think it will add noise, but will add

Comment (oshoval) [reply hack/bump-cnao.sh:19]: Its pretty noisy, it prints the yamls that it download (to a variable)
and so on, you sure we want it ?
can you run locally please (unless you did), see how it looks like and suggest ?

**Comment by oshoval** [comment]
Aligned styles

File: hack/bump-cnao.sh (hunk 1 above)
Comment (RamLavi) [reply hack/bump-cnao.sh:19]: I get you. I won't insist on it if you it brings more than than good

**Review by RamLavi — APPROVED** [review]
/lgtm

**Comment by oshoval** [comment]
failures of the lanes is fixed by 
https://github.com/kubevirt/project-infra/pull/1207

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**